code/modules/cargo/packs/general.dm

```diff
@@ -263,13 +263,15 @@
 	///Total TC worth of contained uplink items
 	var/crate_value = 30
 	var/uplink_flag = UPLINK_TRAITORS
+	///What uplink the contents are pulled from
+	var/contents_uplink_type = UPLINK_TRAITORS
```

Comment:
What's the difference between these two vars? Do we override one somewhere that I'm not seeing? Just seems silly to have two flags corresponding to the same thing.